cmd/remote/cmd.go

```diff
@@ -46,7 +46,7 @@ func mainWithArgs(ctx context.Context, _ []string, logger logging.Logger) error
 		},
 	}
 
-	myRobot, err := robotimpl.New(ctx, conf, logger)
```

Comment:
signature of this function changed in https://github.com/viamrobotics/rdk/pull/4782/files